src/sentry/apidocs/constants.py

```diff
@@ -29,3 +29,6 @@
 
 # 409
 RESPONSE_CONFLICT = OpenApiResponse(description="Conflict")
+
+# 429
+RESPONSE_TOO_MANY_REQUESTS = OpenApiResponse(description="Too Many Requests")
```

Comment:
Not relevant to this endpoint, but a signficant amount of our endpoints are rate limited. It makes sense for this to be a constant.